app/components/Account/index.js

```diff
@@ -3,6 +3,7 @@ import { connect } from 'react-redux'
 import wallet from 'ethereumjs-wallet'
 
 import { createQrImage } from '../../utils/qrdisplay'
+import config from '../../../config'
```

Comment:
What is does the ```config```?

Reply:
This config file allows to set the Safe address and the network. Right now it is only used under development conditions.